route/settings/SAMPLE.control

```diff
@@ -12,77 +12,93 @@
 ! ****************************************************************************************************************************
 ! RUN CONTROL 
 ! --------------------------------------------
-<case_name>             CASE_NAME                  ! name of simulation
-<sim_start>             yyyy-mm-dd hh:mm:ss        ! time of simulation start. hh:mm:ss can be omitted
-<sim_end>               yyyy-mm-dd hh:mm:ss        ! time of simulation end. hh:mm:ss can be omitted 
-<route_opt>             1                          ! option for routing schemes 0->accum 1->IRF (default) 2->KWT 3->KW 4->MC 5->DW 
-<doesBasinRoute>        1                          ! basin routing options: 0-> no, 1->IRF, otherwise error
-<restart_write>         last                       ! restart write options: never, daily, monthly, annual, last, specified 
-<restart_date>          yyyy-mm-dd hh:mm:ss        ! restart date.  activated only if <restart_write> is "specified" 
-<fname_state_in>        INPUT_RESTART_NC           ! input restart netCDF name. remove for run without any particular initial channel states
-<newFileFrequency>      month                      ! output netcdf frequency: single, day, month, annual  
+<case_name>               CASE_NAME                  ! name of simulation
+<sim_start>               yyyy-mm-dd hh:mm:ss        ! time of simulation start. hh:mm:ss can be omitted
+<sim_end>                 yyyy-mm-dd hh:mm:ss        ! time of simulation end. hh:mm:ss can be omitted
+<route_opt>               1                          ! options for routing schemes (multiple options allowed): 1->IRF 2->KWT 3->KW 4->MC 5->DW
+<doesBasinRoute>          1                          ! basin routing options: 0-> no, 1->IRF, otherwise error
+<restart_write>           last                       ! restart write options: never, daily, monthly, yearly, last, specified
+<restart_date>            yyyy-mm-dd hh:mm:ss        ! restart date.  activated only if <restart_write> is "specified"
+<fname_state_in>          INPUT_RESTART_NC           ! input restart netCDF name. remove or 'coldstart' for run without any particular restart file
+<newFileFrequency>        month                      ! output netcdf frequency: single, daily, monthly, yearly
+<dt_qsim>                 86400                      ! time interval of the forcing [sec] e.g., 86400 sec for daily step
+! lake mode
+<is_lake_sim>             T                          ! switch on (T) or off (F) lake simulation
+<is_flux_wm>              T                          ! switch on (T) or off (F) abstraction from or injection to seg/lakes
```

Comment:
These three variables aren't very clear as to what their purpose is. Can you help clarify a bit? I'd also give a reminder that "wm" stands for Water Management

Reply:
modified comment "!lake mode" to "!lake and water-management model"